Doc/library/pathlib.rst

```diff
@@ -136,6 +136,14 @@ we also call *flavours*:
       >>> PurePath()
       PurePosixPath('.')
 
+   An empty string in *pathsegments* also refers to the current directory.
+   This means that ``PurePath("")`` is equivalent to ``PurePath(".")``,
+   and consequently ``Path("").exists()`` will always return ``True``. This
+   differs from ``os.path.exists("")``, which returns ``False``.
+
+      >>> PurePath("")
+      PurePosixPath('.')
```

Comment:
I think the new text should be folded into the previous 4 lines.  The result would then be:
>When *pathsegments* is empty or a single empty string, the current directory is assumed::

      >>> PurePath(), PurePath('')
      (PurePosixPath('.'), PurePosixPath('.'))

The boolean value of either expression is True.
This differs from ``os.path.exists("")``, which returns ``False``.